src/components/AddPodcast/index.js

```diff
@@ -0,0 +1,98 @@
+import {useRouter} from 'next/router';
+import {useState} from 'react';
+
+import useStore from '../../hooks/useStore';
+import loadPodcastFeed from '../../utils/loadPodcastFeed';
+
+import {AddPodcastForm, ButtonRow, AddPodcastNotificaton, NotificationArea} from './styled';
+
+export default function AddPodcast() {
+	const router = useRouter();
+	const [message, setMessage] = useState('form');
+	const [addedPodcastTitle, setAddedPodcastTitle] = useState('');
+	const {addPodcast} = useStore();
+
+	function handleCancel() {
+		router.push('/');
+	}
```

Comment:
You could also write this directly in the `onClick` of the button if you like :)

Reply:
Thanks! That saves some lines of code.